colorfield/fields.py

```diff
@@ -12,28 +12,50 @@
     from django.utils.translation import ugettext_lazy as _
 
 
-COLOR_RE = re.compile('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')
-color_validator = RegexValidator(
-    COLOR_RE, _('Enter a valid color.'), 'invalid')
+COLOR_HEX_RE = re.compile('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')
+color_hex_validator = RegexValidator(
+    COLOR_HEX_RE, _('Enter a valid hex color, eg. #000000'), 'invalid')
+
+COLOR_HEXA_RE = re.compile('#([A-Fa-f0-9]{8}|[A-Fa-f0-9]{4})$')
+color_hexa_validator = RegexValidator(
+    COLOR_HEXA_RE, _('Enter a valid color, eg. #00000000'), 'invalid')
```

Comment:
You forgot to add the format: `Enter a valid hexa color, ...`

Reply:
No problem!

Reply:
✔️